README.md

```diff
@@ -24,8 +24,8 @@ released version of Riot:
    as desired. See below for details.
 1. Enter the URL into your browser and log into Riot!
 
-Releases are signed by PGP, and can be checked against the public key
-at https://riot.im/packages/keys/riot.asc .
+Releases are signed using gpg and the OpenPGP standard, and can be checked against the public key located
+at https://packages.riot.im/debian/riot.im-archive-keyring.asc .
```

Comment:
This doesn't exist, did you mean `riot-im-archive-keyring.asc` ? Also someone is going to have to explain to me the difference between that and https://packages.riot.im/riot-release-key.asc

Reply:
Good catch. Thanks.
I really meant https://packages.riot.im/riot-release-key.asc, I'll amend.

Reply:
given that's also Dave's suggestion, I'm inclined to trust that too :D

Reply:
So we've got https://packages.riot.im/riot-release-key.asc for the general package signing and https://packages.riot.im/debian/riot.im-archive-keyring.asc as the key to add to apt's key store, on Debian. Correct?

Reply:
Those sound like the right things to me